RIOT-OS/RIOT
# Draft: Sneak peek: Gameboy Advance support

### Contribution description

Moo!

This adds the Gameboy Advance to the supported boards for RIOT.
This is an (very) early draft, intended to gather thoughts and feedback.
There is no need for an in-depth review but comments on the integration / api design towards RIOT are very welcome.

### Testing procedure

Go to `examples/hello-world` and run `BOARD=gba_cartridge make` afterwards, plug the `bin/gba_cartridge/hello-world.bin` in an emulator of you choice. Please note that some examples e.g. ipc_pingpong might need the official GBA BIOS to run properly, depending on your emulator.
